opensfm/src/bundle/CMakeLists.txt

```diff
@@ -22,25 +22,27 @@ target_link_libraries(bundle
     ${SUITESPARSE_LIBRARIES}
     foundation
 )
+
 if (LAPACK_FOUND)
     target_include_directories(bundle PRIVATE ${LAPACK_INCLUDE_DIRS})
 endif()
 if (SUITESPARSE_FOUND)
     target_include_directories(bundle PRIVATE ${SUITESPARSE_INCLUDE_DIRS})
 endif()
-target_include_directories(bundle PRIVATE ${CERES_INCLUDE_DIR} ${CMAKE_SOURCE_DIR})
+target_include_directories(bundle PRIVATE ${CMAKE_SOURCE_DIR})
```

Comment:
But otherwise preferring to reference only target names (`Ceres::ceres` in `target_link_libraries`), instead of lists of paths (`CERES_INCLUDE_DIR`)

Reply:
Hello! It would because we keep `target_link_libraries(... ${CERES_LIBRARIES})`, where `${CERES_LIBRARIES}` refer to an exported Ceres cmake target (it propagates the include directories). The ceres/Ceres::ceres target would be discovered by `find_package` through the `CMAKE_PREFIX_PATH` environment variable

Reply:
For example, Nix puts ceres in a "custom" location at `/nix/store/${hash}-ceres-solver/`, but cmake is able to discover it, because with [ceres in "inputs"](https://github.com/NixOS/nixpkgs/blob/c02944e17d0d8ea91c9dc8878dc93697f7098838/pkgs/development/python-modules/opensfm/default.nix#L78) Nix adds this path to `CMAKE_PREFIX_PATH` list